assignments/objects.js

```diff
@@ -18,28 +18,87 @@ const example = {
 }
 
 // Write your intern objects here:
+const Student1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const Student2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const Student3 = {
+  "id":3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+
+const Student4 = {
+  "id":4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+
+}
+
+const Student5 = {
+  "id":5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
+
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(Student1["name"]);
+
 // Kennan's ID
+console.log(Student2["id"]);
 
 // Keven's email
+console.log(Student3.email);
 
 // Gannie's name
+console.log(Student4.name);
 
 // Antonietta's Gender
+console.log(Student5.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+Student3.speak = function (name){
+  return "hi my name is" + " " + Student3.name;
```

Comment:
So you don't need a parameter here. Part of the point of this exercise is to make the method object specific. So you could have the string be hard coded to say `"Hi my name is Kennan"` Instead of passing name in or accessing it on the object.